scripts/india_plfs/wage_data/README.md

```diff
@@ -0,0 +1,70 @@
+# Periodic Labour Force Survey (PLFS) - Average wage/salary earnings
+
+
+## About the Dataset
+Average wage/salary earnings (Rs. 0.00) during the preceding calendar month from regular wage/salaried employment among the regular wage/ salaried employees in CWS for each State/ UT.
+
+### Download URL
+
+* PLES July 2018 - June 2019 [XLSX](http://mospi.nic.in/sites/default/files/reports_and_publication/PLFS_2018_19_Anual/Table_42.xlsx)
```

Comment:
How often is this data released?  Is there data older than this?

(Generally, if the format stays mostly same, its good to get data from as many years as there exists.)

Reply:
Usually once a year. I am trying to find older reports to extract the data. The format seems to be same.
 I added 2017-2018. I will add as I find more.